### ISSUE TITLE
Table environments - Unique attribute combination

### ISSUE THREAD

**Issue by gabrielbouleau** [mangal-interactions/mangal-api#12]
What should be the unique compination in the environment table?

Actually: localisation, date and attr_id

Howerver, some papers have only one spacial reference and date for three plots (three different networks) but different environment values. Shall we remove the only to allow multiple entry for the same localisation/date/attr_id?

**Comment by SteveViss** [comment]
Yes, the constraint should be set on localisation and date (w/o attr_id). Is this will fix the issue @gabrielbouleau ?
